index.html

```diff
@@ -102,7 +102,7 @@ <h2>Tonic</h2>
             />
           </div>
           <div class="card1txt">
-            <h2>Tonic</h2>
+            <h2>Multi-Post Stories</h2>
```

Comment:
:red_circle: [REQUIRED] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. You should loop through your projectContent array and for each item kindly make sure you generate the card that will be append to this projects section.

Reply:
Done